### ISSUE TITLE
Class table and half class table both break away from their block and destroys the page

### ISSUE THREAD

**Issue by guppy42** [stolksdorf/NaturalCrit#49]
**Browser Type/Version: ** Chrome Version 48.0.2564.116 (64-bit)

**Operating System: ** Ubuntu 14.04

**Issue Description** : 

Inserting a class table of half class table after a paragraph means the header of the table gets pushed to the 2nd column. putting it after a wide monster stat block works as intended.

**Markdown code to reproduce: **

http://www.naturalcrit.com/homebrew/edit/N1faI-cbsg

**Images: **

http://i.imgur.com/pPSciUK.png

**Comment by guppy42** [comment]
Realise somebody migth edit the above create a new brew input a headder, and a paragrap like so;
`# Usefull facts

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris. Maecenas congue ligula ac quam viverra nec consectetur ante hendrerit. Donec et mollis dolor. Praesent et diam eget libero egestas mattis sit amet vitae augue. Nam tincidunt congue enim, ut porta lorem lacinia consectetur. Donec ut libero sed arcu vehicula ultricies a non tortor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ut gravida lorem. Ut turpis felis, pulvinar a semper sed, adipiscing id dolor. Pellentesque auctor nisi id magna consequat sagittis. Curabitur dapibus enim sit amet elit pharetra tincidunt feugiat nisl imperdiet. Ut convallis libero in urna ultrices accumsan. Donec sed odio eros. Donec viverra mi quis quam pulvinar at malesuada arcu rhoncus. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In rutrum accumsan ultricies. Mauris vitae nisi at sem facilisis semper ac in est.
`

and then insert a class table

**Comment by stolksdorf** [comment]
Unfortunately there is fix for this without changing many components of the tool. *Technically* this is behaving the way it's suppose to. I designed the class tables to work at the top of a page, as they are always at the top of the page in the PHB. 

**Technical on why this is happening**: A class table has both a h5 title element and a table element. The title element flows with the 2 column layout, while the table does not, it spans both. I fake the look of the title being in the table by using a negative top margin on the table. So in your example the title is flowing along with the two column layout and is rendering on column two, and the table looks like it's bleeding into the rest of the text because of the negative margin pulling it up. 

Going between 2 column rendering and full width is where most of the graphical issues people are having come from. And there really isn't anything to fix, they just don't behave the way people except sometimes.

**TL;DR** Put your class tables at the top of the page.
